test/e2e/components/vl-radio.js

```diff
@@ -0,0 +1,24 @@
+const {VlElement} = require('vl-ui-core').Test;
+const {By} = require('vl-ui-core').Test.Setup;
+
+class VlRadio extends VlElement {
+  constructor(driver, selector) {
+    super(driver, selector);
+  }
+
+  async isChecked() {
+    const input = await this._getInput();
+    return this.driver.executeScript('return arguments[0].checked', input);
+  }
+
+  async toggle() {
```

Comment:
Best altijd de gebruiker emuleren. Hier zou dus een functie moeten komen die aan de hand van de click()-functie van Selenium, een toggle uitvoert.

Reply:
Helaas niet gevonden. als ik de click op de input doe, dan zegt die: 
`     ElementClickInterceptedError: element click intercepted: Element <input class="vl-radio__toggle" type="radio" id="radio" name="group-0" value="yes"> is not clickable at point (40, 379). Other element would receive the click: <div class="demo">...</div>
`